singlestoredb/management/job.py

```diff
@@ -0,0 +1,844 @@
+#!/usr/bin/env python
+"""SingleStoreDB Cloud Scheduled Notebook Job."""
+import datetime
+import time
+from enum import Enum
+from typing import Any
+from typing import Dict
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import Type
+from typing import Union
+
+from ..exceptions import ManagementError
+from .manager import Manager
+from .utils import camel_to_snake
+from .utils import from_datetime
+from .utils import get_cluster_id
+from .utils import get_database_name
+from .utils import get_virtual_workspace_id
+from .utils import get_workspace_id
+from .utils import to_datetime
+from .utils import to_datetime_strict
+from .utils import vars_to_str
+
+
+class ParameterType(Enum):
```

Comment:
Were you still going to remove these and just use strings?

Reply:
I removed the parameter type and used a map instead. The rest we were planning to keep as Enums.
Are you okay with that approach @kesmit13?

Reply:
Yes, we just need to be careful when new values are added to the Management API endpoints to make sure the SDK gets updated and installed in the containers at the same time.